### PR TITLE
add certification to configuration

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: pkg/apis/radixvalidators/validate_ra.go [hunk 1]
```diff
@@ -105,6 +106,10 @@ func CPUResourceRequirementFormatError(value string) error {
 	return fmt.Errorf("Format of cpu resource requirement %s (value %s) is wrong. Must match regex '%s'", "cpu", value, cpuRegex)
 }
 
+func InvalidValidationType(verification string) error {
```
Comment (nilsgstrabo) [inline pkg/apis/radixvalidators/validate_ra.go:109]: Name should be InvalidVerificationType

File: pkg/apis/radixvalidators/validate_ra_test.go [hunk 2]
```diff
@@ -155,19 +155,19 @@ func Test_invalid_ra(t *testing.T) {
 		{"too long environment variable name", radixvalidators.InvalidStringValueMaxLengthError("environment variable name", wayTooLongName, 253), func(ra *v1.RadixApplication) {
 			ra.Spec.Components[1].EnvironmentConfig[0].Variables[wayTooLongName] = "Any value"
 		}},
-		{"conflicting variable and secret name", radixvalidators.SecretNameConflictsWithEnvironmentVariable(validRASecondComponentName, conflicingVariableName), func(ra *v1.RadixApplication) {
```
Comment (nilsgstrabo) [inline pkg/apis/radixvalidators/validate_ra_test.go:158]: I think we should add authentication config to the radixconfig.yaml used to test a valid RA (in line 97)
image

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Looks good! Just two minor changes that needs to be adressed

File: pkg/apis/radixvalidators/validate_ra.go (hunk 1 above)
Comment (Pespiri) [reply pkg/apis/radixvalidators/validate_ra.go]: Fixed

File: pkg/apis/radixvalidators/validate_ra_test.go (hunk 2 above)
Comment (Pespiri) [reply pkg/apis/radixvalidators/validate_ra_test.go:158]: Added

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**